ledger/ledger-api-test-tool-on-canton/canton.conf

```diff
@@ -2,14 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 canton {
-  parameters {
-    participant {
-      party-change-notification {
-        type = via-domain
-      }
-    }
-  }
```

Comment:
These canton.parameters.participant are now specified per participant, but "via-domain" is the default party-change-notification type anyway, so it seemed cleaner to just remove this.